### PR TITLE
chore: reorganize common Kafka types and labels

### PR DESCRIPTION
**Pull request by riccardo-forina**
This reorganize and simplifies the types that are common between the various Kafka components a bit

### CODE REVIEW

File: locales/en/kafka.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "reset_offset": "Reset offset",
     "reset_offset_accept": "I understand that by resetting the offset position, I risk clients skipping or duplicating messages.",
     "reset_offset_connected_alert_body": "One or more members of the consumer group are currently connected to a topic. All members of a consumer group must be disconnected before resetting the offset.",
-    "reset_offset_connected_alert_title": "The offset for this consumer group cannot be reset at this time",
+    "reset_offset_connected_alert_title": "The offset for this consumer group caénnot be reset at this time",
```
Comment (HemaHG) [inline locales/en/kafka.json:76]: Is caennot a typo?

Comment (riccardo-forina) [reply locales/en/kafka.json:76]: yep, not sure how it did end there. Thanks for checking!

File: locales/en/kafka.json [hunk 2]
```diff
@@ -139,8 +156,38 @@
...
+    "actions": {
+      "change-owner": "Change owner",
+      "connection": "Connection",
+      "delete": "Delete instance",
+      "details": "Details",
+      "view-instance-information": "View instance information"
+    },
+    "title": "Kafka Instances"
+  },
+  "TODO": "",
```
Comment (HemaHG) [inline locales/en/kafka.json:190]: I guess we don't need this

**Review by HemaHG — APPROVED** [review]
LGTM

**Pull request merged**